src/components/CountryCard/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import CustomCheckbox from '../customCheckbox/index';
+import './styles.scss'
+
+const CountryCard  = ({country, onChange, deleteHandler, path}) => {
+    return (
+        <Link to={`${path}`} >
+            <div className='card-wrapper'>
```

Comment:
You've created styled  flex block  you could extend it with paddings and other styles to re-use it and not use css styling with classNames